05_syntax_lab/06.py

```diff
@@ -1,9 +1,11 @@
-"""
-Write a program that randomizes 2 numbers
-and calculates their least common multiplier,
-that is the smallest number that is divisable
-by both.
-For example if the numbers were 4 and 6,
-program should print 12
-"""
-
+from random import randint
+import sys
+
+x = randint(1,10)
+y = randint(1,10)
+print 'x: ', x, 'y: ', y
+for ix in range(1, x+1):
+    for iy in range(1, y+1):
+        if y * ix == x * iy:
+            print y * ix
+            sys.exit(0)
```

Comment:
Python has a gcd function built-in (read here: https://docs.python.org/2/library/fractions.html#fractions.gcd)

Can you use that to simplify your solution?